src/Apps/Artist/Components/ArtistConsignButton.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ArtistConsignButton_artist } from "__generated__/ArtistConsignButton_artist.graphql"
+import { RouterLink } from "Artsy/Router/RouterLink"
+import { Media } from "Utils/Responsive"
+import { getConsignmentData } from "../Routes/Consign/Utils/getConsignmentData"
+
+import {
+  BorderBox,
+  Box,
+  Button,
+  color,
+  Flex,
+  Image,
+  Sans,
+} from "@artsy/palette"
+
+export interface ArtistConsignButtonProps {
+  artist: ArtistConsignButton_artist
+}
+
+export const ArtistConsignButton: React.FC<ArtistConsignButtonProps> = props => {
+  return (
+    <>
+      <Media at="xs">
+        <ArtistConsignButtonSmall {...props} />
+      </Media>
+      <Media greaterThan="xs">
+        <ArtistConsignButtonLarge {...props} />
+      </Media>
+    </>
+  )
+}
+
+export const ArtistConsignButtonLarge: React.FC<ArtistConsignButtonProps> = props => {
+  const { isTop20, imageURL, headline, consignLink } = getData(props)
+
+  return (
+    <BorderBox p={1} width="100%">
+      <Flex alignItems="center" width="100%" justifyContent="space-between">
+        <Flex>
+          {isTop20 && <Image src={imageURL} width={50} height={50} />}
+          <Flex flexDirection="column" justifyContent="center" pl={1}>
+            <Sans size="3t" weight="medium">
+              {headline}
+            </Sans>
+            <Box position="relative">
+              <Sans size="3t" color={color("black60")}>
+                Consign with Artsy
+              </Sans>
+            </Box>
+          </Flex>
+        </Flex>
+        <Box>
+          <RouterLink to={consignLink}>
+            <Button variant="secondaryGray">Get started</Button>
+          </RouterLink>
+        </Box>
+      </Flex>
+    </BorderBox>
+  )
+}
+
+export const ArtistConsignButtonSmall: React.FC<ArtistConsignButtonProps> = props => {
+  const { isTop20, imageURL, headline, consignLink } = getData(props)
+
+  return (
+    <BorderBox maxWidth={335} p={1}>
+      <Flex alignItems="center">
+        {isTop20 && <Image src={imageURL} width={75} height={66} />}
+        <Flex flexDirection="column" justifyContent="center" pl={1}>
+          <Sans size="3t" weight="medium">
+            {headline}
+          </Sans>
+          <Box top="-2px" position="relative">
+            <Sans size="3t" color={color("black60")}>
+              Consign with Artsy
+            </Sans>
+          </Box>
+          <Box>
+            <RouterLink to={consignLink}>
+              <Button size="small" variant="secondaryGray">
+                Get started
+              </Button>
+            </RouterLink>
+          </Box>
+        </Flex>
+      </Flex>
+    </BorderBox>
+  )
+}
+
+function getData(props) {
+  const {
+    artist: { href, name, image },
+  } = props
+  const isTop20 = Boolean(getConsignmentData(href))
```

Comment:
Around the time we we address the Redshift data issue I'll move this CSV json functionality into Convection or MP GraphQL layer where we can retrieve it via API request.

Reply:
May I suggest slightly different naming here? The `top20` seems like it will become outdated/confusing soon (I've already heard talks of how these artists aren't really "top"). Can we refer more to the feature at-hand? `hasRichConsignContent` (that's not great, but hopefully you get the idea 😅)

Reply:
Sure thing - this was referenced in design docs so I'll raise in consign channel 👍